.github/label-actions.yml

```diff
@@ -5,13 +5,92 @@
 #   - https://github.com/dessant/label-actions/issues/1
 pulls:
   actions:
+    attic:
+      close: true
+      comment: |
+        Thanks for your interest in contributing to the Metasploit Framework! For now it seems that this pull request isn't quite ready to land, and we have opted to close and tag the pull request with the `attic` label.
+
+        What does this generally mean?
+        
+        - It doesn't look like there's been any activity on this Pull Request in quite some time
+        - We or the contributor doesn't have the time or equipment necessary to test it or fix it up
+        - Sometimes the implementation isn't quite right and a different approach is necessary
+        
+        We would love to land this pull request when it's ready. If you have a chance to address the comments, we'd be happy to reopen and discuss merging this contribution into the framework! :tada:
+        
     needs-docs:
       comment: |
-        Thanks for your pull request, before this can be merged - corresponding documentation for your module is required:
+        Thanks for your pull request! Before this can be merged, we need the following documentation for your module:
+
+        - [Writing Module Documentation](https://github.com/rapid7/metasploit-framework/wiki/Writing-Module-Documentation)
+        - [Template](https://github.com/rapid7/metasploit-framework/blob/master/documentation/modules/module_doc_template.md)
+        - [Examples](https://github.com/rapid7/metasploit-framework/tree/master/documentation/modules)
+
+    needs-linting:
+      comment: |
+        Thanks for your pull request! Before this pull request can be merged, it must pass the checks of our automated linting tools.
+        
+        We use Rubocop and msftidy to ensure the quality of our code. This can be ran from the root directory of Metasploit:
+        
+        ```
+        rubocop <directory or file>
+        tools/dev/msftidy.rb <directory or file>
+        ```
+        
+        You can automate most of these changes with the `-a` flag:
+        
+        ```
+        rubocop -a <directory or file>
+        ```
+        
+        Please update your branch after these have been made, and reach out if you have any problems.
```

Comment:
![image](https://user-images.githubusercontent.com/60357436/79617633-db26f280-80ff-11ea-9559-9ce9105ae667.png)

Reply:
Love it!